.github/workflows/cicd.yml

```diff
@@ -68,7 +70,7 @@ jobs:
   unit_tests:
     # Always run this job.
     needs: regular
-    if: github.event_name == 'schedule' || github.event_name == 'pull_request' || github.event_name == 'push' || github.event_name == 'merge_group'
+    if: github.event_name == 'schedule' || github.event_name == 'pull_request' || github.event_name == 'push' || github.event_name == 'merge_group' || github.event_name == 'workflow_dispatch'
```

Comment:
added `workflow_dispatch` for all jobs which run nightly on schedule, not those which only run at PR time.

Reply:
I'd agree.